### ISSUE TITLE
notebook 05_TensorFlow Error

### ISSUE THREAD

**Issue by NLGRF** [madewithml/lessons#176]
The topic "Gradients" is PyTorch no TensorFlow

**Comment by GokuMohandas** [comment]
uh oh good catch @NLGRF , sometimes I just want to write in both frameworks :)
